client-v2/src/main/java/com/clickhouse/client/api/ClickHouseSettings.java

```diff
@@ -0,0 +1,56 @@
+package com.clickhouse.client.api;
+
+import com.clickhouse.data.ClickHouseFormat;
+
+import java.util.*;
+
+public class ClickHouseSettings {
```

Comment:
Lets do not use `ClickHouse` prefix in classes because it makes names longer. 
We would need it where our classes may appear.